### DISCUSSION TITLE
Can i only partially use this for my ipad, but not android and windows?

### DISCUSSION THREAD

**Discussion by FireRedDev** [remotely-save/remotely-save#786] [Q&A]
I use OneDrive and OneSync on android and would like to use this for my ipad without adding it to other devices, is that possible?

**Comment by fyears** [comment]
firstly i strongly recommend you backup your vault elsewhere, or use a test vault to try.

secondly i don't recommend doing so.

thirdly you need to pay attention to the location in onedrive website. currently the plugin only allows syncing to a folder in the root or a folder in /Apps , so you want to place the vault folder accordingly.
